backend/api/v1/endpoints/Research/UploadPaper.py

```
@@ -0,0 +1,67 @@
+        # ✅ Save file to disk
+        save_upload_file(file, UPLOAD_DIR, safe_filename)
+
+        # ✅ Save metadata to DB
+        new_paper = ResearchPaper(
+            title=title,
+            author=author,
+            research_field=research_field,
+            file_path=safe_filename,  # Only the filename
+            original_filename=file.filename,
+            uploader_id=current_user.id,
+            created_at=datetime.now(timezone.utc)
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)